controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -236,7 +236,19 @@ http {
         ssl_verify_client on;
         ssl_verify_depth {{ $location.CertificateAuth.ValidationDepth }};
         {{ end }}
+        
+        {{ if (or $location.Redirect.ForceSSLRedirect (and (not (empty $server.SSLCertificate)) $location.Redirect.SSLRedirect)) }}
```

Comment:
@aledbf I've only changed the piece of code from it's location. The idea is first make it's HTTPs redirection based on the configs (as defined previous in lines 282:287 and then moved to this new location) and then redirect (if it exists) to the root context.

One doesn't exclude other, this code existed previously. It only orders which rules will be followed first :)